### PR TITLE
Complete NumPy ufunc compatibility in Series

### PR DESCRIPTION
**Pull request by HyukjinKwon**
This PR completes NumPy's ufunc support (followup of https://github.com/databricks/koalas/pull/1096).

See also https://docs.scipy.org/doc/numpy/reference/arrays.classes.html#standard-array-subclasses

E.g.:

```python
>>> import databricks.koalas as ks
>>> import numpy as np
>>> kdf = ks.range(10)
>>> kser = np.sqrt(kdf.id)
>>> type(kser)
<class 'databricks.koalas.series.Series'>
>>> kser
0    0.000000
1    1.000000
2    1.414214
3    1.732051
4    2.000000
5    2.236068
6    2.449490
7    2.645751
8    2.828427
9    3.000000
```

### CODE REVIEW

File: databricks/koalas/tests/test_numpy_compat.py [hunk 1]
```diff
@@ -43,6 +45,12 @@ def test_np_add_series(self):
         pdf = self.pdf
         self.assert_eq(np.add(kdf.a, 1), np.add(pdf.a, 1))
 
+    def test_np_abs_series(self):
+        kdf = self.kdf
+        pdf = self.pdf
+        np.abs(kdf.a)
```
Comment (itholic) [inline databricks/koalas/tests/test_numpy_compat.py]: this line for what? 😮

File: databricks/koalas/tests/test_numpy_compat.py [hunk 2]
```diff
@@ -52,3 +60,44 @@ def test_np_unsupported(self):
...
+            np.random.randint(-100, 100, size=(np.random.randint(100), 2)), columns=['a', 'b'])
+        kdf = ks.from_pandas(pdf)
+
+        blacklist = [
+            # Koalas does not currently support
+            "conj",
+            "conjugate",
+            "isnat",
+            'matmul',
+            'frexp',
```
Comment (itholic) [inline databricks/koalas/tests/test_numpy_compat.py]: i wonder why `'matmul'` and `'frexp'` are only using single quote ??

is there any special purpose??

Comment (HyukjinKwon) [reply databricks/koalas/tests/test_numpy_compat.py]: Thanks, I will fix them.

**Comment by HyukjinKwon** [comment]
tests passed.

File: databricks/koalas/tests/test_numpy_compat.py [hunk 3]
```diff
@@ -52,3 +52,54 @@ def test_np_unsupported(self):
...
         with self.assertRaisesRegex(NotImplementedError, "Koalas.*not.*support.*sqrt.*"):
             np.sqrt(kdf.a, kdf.b)
+
+    def test_np_spark_compat(self):
+        # Use randomly generated dataFrame
+        pdf = pd.DataFrame(
+            np.random.randint(-100, 100, size=(np.random.randint(100), 2)), columns=['a', 'b'])
+        kdf = ks.from_pandas(pdf)
+
+        blacklist = [
```
Comment (HyukjinKwon) [inline databricks/koalas/tests/test_numpy_compat.py]: Btw, I ran it few hundreds times and it seems not flaky.

**Comment by ueshin** [comment]
Thanks! merging.

**Pull request merged**